ALMA-FUNDEGUA/vacunasgt
# ALMAT-1057

# ALMAT-1057
​
## Description
- [X] Minor change

Changed preview information including:
- Instagram/Facebook preview image
- Page title and website
​
## Manual testing
- How to verify:
  - 1. Copy VacunasGT link
  - 2. Share link to any social media

​
## Images:
![image](https://user-images.githubusercontent.com/60187741/216207654-1bbfb3ef-2f60-408d-b60d-d20bfb43a2ad.png)
